docs/product/components/tables.html

```diff
@@ -446,40 +507,332 @@
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    </div>
 
-    {% header {element: 'h3', text: 'Totals Row'} %}
-    <p class="stacks-copy">Used mainly with data tables, the totals row increases the font-size for all cells within a row.</p>
+    {% header {element: 'h3', text: 'Text Alignment'} %}
+    <p class="stacks-copy">Text alignment can be changed at a table or cell level by using <a href="{{ "/product/base/typography#layout-classes" | relative_url }}">atomic text alignment classes</a>. Columns containing copy should be left-aligned. Columns containing numbers should be right-aligned.</p>
     <div class="stacks-preview">
 {% highlight html linenos %}
+<table class="s-table">
+    <tbody>
+        <tr>
+            <td class="ta-left">…</td>
+            <td class="ta-center">…</td>
+            <td class="ta-right">…</td>
+        </tr>
+    </tbody>
+</table>
 <table class="s-table ta-right">
+    <tbody>
+        <tr>
+            <td>…</td>
+            <td>…</td>
+            <td>…</td>
+        </tr>
+    </tbody>
+</table>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <table class="s-table mb16">
+                <tbody>
+                    <tr>
+                        <td class="stacks-grid-example--cell ta-left hs1"><code class="stacks-code">.ta-left</code></td>
+                        <td class="stacks-grid-example--cell ta-center"><code class="stacks-code">.ta-center</code></td>
+                        <td class="stacks-grid-example--cell ta-right"><code class="stacks-code">.ta-right</code></td>
+                    </tr>
+                </tbody>
+            </table>
+            <table class="s-table ta-right">
+                <tbody>
+                    <tr>
+                        <td class="stacks-grid-example--cell hs1"><code class="stacks-code">.s-table.ta-right</code></td>
+                        <td class="stacks-grid-example--cell"><code class="stacks-code">.s-table.ta-right</code></td>
+                        <td class="stacks-grid-example--cell"><code class="stacks-code">.s-table.ta-right</code></td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    </div>
+</section>
+<section class="stacks-section">
+    {% header {element: 'h2', text: 'Sortable Tables'} %}
+    <p class="stacks-copy">With the addition of JavaScript, table columns can be re-sorted based on the columns data.</p>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<table class="s-table s-table__sortable">
     <thead>
         <tr>
-            <th scope="col" class="ta-left">Listing</th>
-            <th scope="col" class="s-table--cell2">Views</th>
-            <th scope="col" class="s-table--cell2">Apps</th>
-            <th scope="col" class="s-table--cell2">App CTR</th>
+            <th scope="col" class="is-sorted">Listing @Svg.ArrowDownSm</th>
```

Comment:
Should we use any `@Svg.xxx` in Stacks code? It's dependent on our backend right now so feels like kinda _dependency_, and since Stacks supposed to be public, maybe we should skip it as well? Or include similar banner as on [Icons page](https://deploy-preview-129--stacks.netlify.com/product/resources/icons).

Reply:
The examples use the liquid Svg helper noted on the Icons page. The primary audience for Stacks is SO employees, so I see no problem listing the SVG helper as we use it in the product within our code snippets. Also this is a pattern we use throughout Stacks.